### DISCUSSION TITLE
مشکل در ریست ترافیک یوزرها

### DISCUSSION THREAD

**Discussion by hayousef68** [Gozargah/Marzban#1063] [General]
سلام از دیروز خیلی از یوزرهام که پیام میدن برای تمدید نمیتونم ترافیک رو ریست کنم خطا میده ( بازنشانی مصرف انجام نشد، دوباره تلاش کنید)

سرور رو عوض کردن مرزبان رو ریستارت کردم بازم درست نشد میشه راهنمایی کنین
![8eeddc35-47b5-45ee-b72e-abd312e2f523](https://github.com/Gozargah/Marzban/assets/144268484/d9c9e2df-7e42-455b-ac19-74e56a6e4502)
.

**Comment by M3h64** [comment]
چند نفر دیگه هم میگفتن این مشکلو دارن. منم همین مشکل رو دارم. البته تو یوزرهایی که قبلا ایجاد شده. یوزرهایی که جدید ایجاد میشن این مشکل رو ندارن. کسی هم پاسخگو نیست

**Comment by waxilu** [comment]
لطفا ابتدا از دستور amrzban update استفاده کنید احتمالا مشکل حل میشه
اما اگر مشکل رفع نشد یوزرا به سرور جدید منتقل کنید
